src/module/LayerZeroModule.sol

```diff
@@ -60,14 +66,16 @@ contract LayerZeroModule is Admin, Initializable, CrossChainMessageInterface, La
       address bridge,
       address interfaces,
       address operator,
+      address optimismGasPriceOracle,
```

Comment:
This is what you mentioned using Uniswap / Sushiswap for right?

Reply:
No this is for calculating Optimism L1 Security Fee